Error: Cannot find module 'lmdb-store'

I'm getting this error with the Gatsby 4.8.0

Gatsby 4.8.0 has upgrade from `lmdb-store` to `lmdb`

```
Error: Cannot find module 'lmdb-store'
Require stack:
- /opt/build/repo/.netlify/plugins/node_modules/@netlify/plugin-gatsby/lib/helpers/files.js
- /opt/build/repo/.netlify/plugins/node_modules/@netlify/plugin-gatsby/lib/index.js
```

--- a/plugin/src/helpers/files.ts
+++ b/plugin/src/helpers/files.ts
@@ -59,8 +59,8 @@ export const findModuleFromBase = ({ paths, candidates }): string | null => {
       if (modulePath) {
         return dirname(modulePath)
       }
-    } catch (error) {
-      console.error(error)
+    } catch {
+      // Ignore error
     }
   }
   return null

